docs/user/openstack/customization.md

```diff
@@ -39,7 +39,7 @@ Beyond the [platform-agnostic `install-config.yaml` properties](../customization
 
 * `additionalNetworkIDs` (optional list of strings): IDs of additional networks for machines.
 * `additionalSecurityGroupIDs` (optional list of strings): IDs of additional security groups for machines.
-* `serverGroupPolicy` (optional string): Server group policy to apply to the group that will contain the machines in the pool. Defaults to "soft-anti-affinity". Only applicable to the Control plane MachinePool.
+* `serverGroupPolicy` (optional string): Server group policy to apply to the group that will contain the machines in the pool. Defaults to "soft-anti-affinity". Note that a strict "affinity" policy prevents migrations, and therefore affects OpenStack upgrades. Allowed values are "affinity", "soft-affinity", "anti-affinity", "soft-anti-affinity".
```

Comment:
thanks for adding "Note that a strict "affinity" policy prevents migrations, and therefore affects OpenStack upgrades."

the other caveat that you may want to consider noting is that this cannot be changed after an instance is created so  if you want to change the member ship of server groups or the policy you will have to destroy the exstign workers and recretate the new ones with the new policy.

you can do that in a rolling fashion if you need too but the point being that the server group membership and policy for any worker instnace is fixed when its created for the lifetime of that worker.

Reply:
Added a note mentioning that a server group is for life. Thanks!